WordPress/gutenberg
# Convert a Classic Block into Multiple Blocks

## Description
See #3149.

This PR adds a button to the right side dropdown for freeform blocks to convert it into separate blocks.

## How Has This Been Tested?
1. Create a Freeform block.
2. Click the right menu, then 'Convert to blocks'.
3. The content of the block should now be parsed into normal blocks.

Thoughts:
* We might want to rename the handler to something more general, like 'rawHandler' or 'convertStringToBlocks', or maybe a separate function that can be used by pasteHandler too.
